app/src/main/java/org/kiwix/kiwixmobile/utils/files/FileUtils.java

```diff
@@ -25,22 +25,18 @@
 import android.os.Environment;
 import android.provider.DocumentsContract;
 import android.util.Log;
-
-import org.kiwix.kiwixmobile.BuildConfig;
-import org.kiwix.kiwixmobile.downloader.ChunkUtils;
-import org.kiwix.kiwixmobile.library.entity.LibraryNetworkEntity.Book;
-
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
+import org.kiwix.kiwixmobile.BuildConfig;
 
 import static org.kiwix.kiwixmobile.downloader.ChunkUtils.deleteAllParts;
 import static org.kiwix.kiwixmobile.utils.Constants.TAG_KIWIX;
 
 public class FileUtils {
+  private static final String TAG = "FileUtils";
```

Comment:
My suggestion is for ALL TAGs to at least start with the word kiwix as this makes them easier to find when filtering and/or reviewing Android adb logs.

Reply:
Done.